xml/System.Text.Json/JsonTokenType.xml

```diff
@@ -94,7 +94,7 @@
       </ReturnValue>
       <MemberValue>9</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The token type is the JSON literal "false".</summary>
```

Comment:
This is tricky... it isn't the literal `"false"` (which is a string), it's the literal `false` (which is a false, since JSON doesn't really have a boolean type per se).  So this ideally should be unquoted and in a different display mode... but I'll go with whatever Ron / Maira think is best.

Reply:
Perhaps remove the quotation marks and boldface it (**false**), @bartonjs?

Reply:
> Perhaps remove the quotation marks and boldface it (false), @bartonjs?

Seems fine to me, not that I know what syntax that would be in a summary tag :smile:.

Reply:
@rpetrusha can we use html tags?

> `<b>false</b>`

Reply:
Yes, we can use HTML tags.